canvas_modules/harness/cypress.config.js

```diff
@@ -12,13 +12,17 @@ module.exports = defineConfig({
 	e2e: {
```

Comment:
can you add this file to be checked in linting.  It should have flagged the new code since we don't use single quotes.